TechDraw build issue on weekly 20260513 with opensuse

### Area / Workbench affected

TechDraw

### Problem description

weekly snapshot 20260513 is seeing a build failure in techdraw module
```
[ 1662s] /home/abuild/rpmbuild/BUILD/FreeCAD-weekly-20260513-build/FreeCAD-weekly-20260513/src/Mod/TechDraw/App/Geometry.cpp: In static member function ‘static std::string TechDraw::GeometryUtils::getGeomTypeName(TechDraw::GeomType)’:
[ 1662s] /home/abuild/rpmbuild/BUILD/FreeCAD-weekly-20260513-build/FreeCAD-weekly-20260513/src/Mod/TechDraw/App/Geometry.cpp:1896:1: error: control reaches end of non-void function [-Werror=return-type]
[ 1662s]  1896 | }
[ 1662s]       | ^
[ 1662s] cc1plus: some warnings being treated as errors
```
looks to be due to a missing default: for the switch statement, adding it does let it compile
it is noted that i was left out: https://github.com/FreeCAD/FreeCAD/blob/751be50f8783ea8ac39b811c90449db8cdd2ec30/src/Mod/TechDraw/App/Geometry.cpp#L1882

### Impact

build failures.

### Suggested improvement (optional)

_No response_

### References / evidence

_No response_

### Development version (if relevant)

```shell

```

--- a/src/Mod/TechDraw/App/Geometry.cpp
+++ b/src/Mod/TechDraw/App/Geometry.cpp
@@ -1879,8 +1879,8 @@ std::vector<int> GeometryUtils::findNestedFaceIndices(const std::vector<FacePtr>
     return nestedFaceIndices;
 }
 
-//! get a description for a GeomType.  Do not add a default case, as we want to
-//! ensure that we are always in sync with the GeomType enum.
+//! get a description for a GeomType.  Needs to always be in sync with the
+//! GeomType enum.
 std::string GeometryUtils::getGeomTypeName(GeomType typeEnumValue)
 {
     switch (typeEnumValue) {
@@ -1893,6 +1893,8 @@ std::string GeometryUtils::getGeomTypeName(GeomType typeEnumValue)
         case GeomType::BSPLINE: return "B-spline Curve";
         case GeomType::GENERIC: return "Line";
     }
+
+    return "Not Defined";
 }
 
 

